sql/branch.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE amount = 0 AND due != 0;
+    UPDATE payment_instructions SET due = floor(9.41/(amount)) * amount WHERE due > 9.41;
+    CREATE TABLE settings (minimum_charge numeric(35,2) DEFAULT NULL);
+    INSERT INTO settings DEFAULT VALUES;
```

Comment:
I guess this `INSERT` is redundant now that `gratipay.wireup.settings` deletes and inserts.